source/mongodump.txt

```diff
@@ -627,17 +627,27 @@ Options
    Creates a file named :file:`oplog.bson` as part of the
    ``mongodump`` output. The ``oplog.bson`` file, located in
    the top level of the output directory, contains oplog entries that
-   occur during the ``mongodump`` operation. This file provides
-   an effective point-in-time snapshot of the state of a
-   ``mongod`` instance. To restore to a specific point-in-time
-   backup, use the output created with this option in conjunction with
-   :option:`mongorestore --oplogReplay`.
+   occur during the ``mongodump`` operation.
+   
+   The ``oplog.bson`` file is a snapshot of the data at the time the
+   file was created. To restore the data as it was when the
```

Comment:
I think it would be more precise to say something like, "To apply those oplog entries during the restore, use mongorestore --oplogReplay. Using mongodump --oplog in conjunction with mongorestore --oplogReplay ensures the data is up-to-date with all writes that occurred during the dump. However, if any writes occur after mongodump completes, those will not be captured in that dump."